2021-08-26 00:53:23,149 INFO    MainThread:28033 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 00:53:23,149 INFO    MainThread:28033 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 00:53:23,150 INFO    MainThread:28033 [wandb_init.py:_log_setup():342] Logging user logs to /home/vicc/neural-sculpture/wandb/run-20210826_005323-25h4932m/logs/debug.log
2021-08-26 00:53:23,150 INFO    MainThread:28033 [wandb_init.py:_log_setup():343] Logging internal logs to /home/vicc/neural-sculpture/wandb/run-20210826_005323-25h4932m/logs/debug-internal.log
2021-08-26 00:53:23,150 INFO    MainThread:28033 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f2e742abf50>
2021-08-26 00:53:23,151 INFO    MainThread:28033 [wandb_init.py:init():375] calling init triggers
2021-08-26 00:53:23,151 INFO    MainThread:28033 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.03, 'batch_size': 16, 'init_tolerance': 0, 'iters_per_res': 16, 'max_iters_per_cam': 32, 'camera': namespace(init_num_cameras=16, mapping_span=6.283185307179586, mapping_type='sdfdiff', max_num_cameras=32, shuffle_order=True), 'loss': namespace(image_loss_weight=0.0002, lp_loss_weight=0.001, sdf_loss_weight=0.001)}
2021-08-26 00:53:23,151 INFO    MainThread:28033 [wandb_init.py:init():424] starting backend
2021-08-26 00:53:23,151 INFO    MainThread:28033 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 00:53:23,175 INFO    MainThread:28033 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 00:53:23,274 INFO    MainThread:28033 [backend.py:ensure_launched():140] started backend process with pid: 28078
2021-08-26 00:53:23,276 INFO    MainThread:28033 [wandb_init.py:init():429] backend started and connected
2021-08-26 00:53:23,279 INFO    MainThread:28033 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 00:53:23,279 INFO    MainThread:28033 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 00:53:23,279 INFO    MainThread:28033 [wandb_init.py:init():477] updated telemetry
2021-08-26 00:53:23,280 INFO    MainThread:28033 [wandb_init.py:init():500] communicating current version
2021-08-26 00:53:23,808 INFO    MainThread:28033 [wandb_init.py:init():505] got version response 
2021-08-26 00:53:23,808 INFO    MainThread:28033 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 00:53:23,971 INFO    MainThread:28033 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 00:53:24,042 INFO    MainThread:28033 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 00:53:24,043 INFO    MainThread:28033 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 00:53:24,043 INFO    MainThread:28033 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 00:53:24,046 INFO    MainThread:28033 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 00:53:24,047 INFO    MainThread:28033 [wandb_init.py:init():565] run started, returning control to user process
2021-08-26 00:53:35,331 INFO    MainThread:28033 [wandb_init.py:_pause_backend():256] pausing backend
2021-08-26 00:53:35,331 INFO    MainThread:28033 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 00:56:58,751 INFO    MainThread:28033 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-08-26 00:56:58,752 INFO    MainThread:28033 [wandb_run.py:_restore():1526] restore
2021-08-26 00:56:59,312 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 833
  total_bytes: 833
}

2021-08-26 00:57:00,020 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 833
  total_bytes: 4354
}

2021-08-26 00:57:00,122 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 833
  total_bytes: 5144
}

2021-08-26 00:57:00,223 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

2021-08-26 00:57:00,325 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

2021-08-26 00:57:00,426 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

2021-08-26 00:57:00,528 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

2021-08-26 00:57:00,740 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

2021-08-26 00:57:00,842 INFO    MainThread:28033 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5144
  total_bytes: 5144
}

